app/Http/Controllers/Volunteer/ChallengeController.php

```diff
@@ -41,6 +41,12 @@ public function create()
      */
     public function store(Request $request)
     {
+        $input = $request->all();
+        $num = $input['events'];
+        if($num<0)
+        {
+            return redirect('volunteer/challenge/create');
+        }
```

Comment:
This should be a validation in the request, and return feedback stating that the number of events should be positive.
Also HTML validation should be added to the tag with minimum value = 0.